src/components/WebexAvatar/WebexAvatar.js

```diff
@@ -1,6 +1,41 @@
-import React from 'react';
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
 import {Avatar} from '@momentum-ui/react';
 
-export default function WebexAvatar(props) {
-  return <Avatar {...props} />;
+const propTypes = {
+  personID: PropTypes.string.isRequired,
+  adapter: PropTypes.object.isRequired,
+};
+
+export default class WebexAvatar extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.props = props;
+    this.state = {
+      person: {},
+    };
+  }
+
+  componentDidMount() {
+    this.props.adapter.getPerson(this.props.personID).subscribe(
```

Comment:
Can we structure our subscribe params into a shape that is a little more readable? I don't remember the order of `next`, `error`, `complete`. The [docs from RxJS](https://rxjs-dev.firebaseapp.com/guide/operators) has a good example that I find easier to process in my head:

```js
  observable.subscribe({
      next(value) {
        const timerID = setTimeout(() => {
          observer.next(value);
          allTimerIDs.delete(timerID);
        }, delayInMillis);
        allTimerIDs.add(timerID);
      },
      error(err) {
        observer.error(err);
      },
      complete() {
        observer.complete();
      }
    });
```

Reply:
so the cool thing about `anonymous` arrow function is that `this` is not defined and mapped to the parent `this` which is what we need for `this.setState` function to work properly.

Reply:
Could we do something like this then?:

```js
const onNext = (person) => {dostuff();};
const onError = (error) => {dostuff();};

this.props.adapter.getPerson(this.props.personID).subscribe(onNext, onError);
```

Just trying to come up with a way to make this as readable as possible for external collaborators.